lib/OnyxUtils.js

```diff
@@ -311,6 +314,21 @@ function isSafeEvictionKey(testKey) {
     return _.some(evictionAllowList, (key) => isKeyMatch(key, testKey));
 }
 
+function storeKeyByConnections(key, connectionID) {
+    if (!onyxKeyToConnectionIDs.has(key)) {
+        onyxKeyToConnectionIDs.set(key, []);
+    }
```

Comment:
This is simply setting a default array right? I miss-read this initially as it looked like we were overwriting the value with `[]`

Reply:
Yes, it is setting a default array